### PR TITLE
Better async when adding components & blocks

### PR DESCRIPTION
**Pull request by rohmanhm**
With this in place, we don't need `async` package anymore. So we can keep saving the bundle size and it will process asynchronously all request without concurrency.

### CODE REVIEW

File: example/my-builder.js [hunk 1]
```diff
@@ -9,22 +9,25 @@ uno.builder('#uno-builder').loadCanvas({
    */
 
   // Basic Blocks
-  uno.addBlock('/example/blocks/section/block.js')
-  uno.addBlock('/example/blocks/section-container/block.js')
+  uno.addBlock([
```
Comment (rohmanhm) [inline example/my-builder.js]: User just write one function and passed an array to it args value, and keep their code be Clean

File: example/my-builder.js [hunk 2]
```diff
@@ -9,22 +9,25 @@ uno.builder('#uno-builder').loadCanvas({
...
-  uno.addComponent('/example/components/column-6/component.js')
-
-  // Basic component
-  uno.addComponent('/example/components/text/component.js')
-  uno.addComponent('/example/components/heading/component.js')
-  uno.addComponent('/example/components/button/component.js')
-  uno.addComponent('/example/components/image/component.js')
-  uno.addComponent('/example/components/video/component.js')
-  uno.addComponent('/example/components/map/component.js')
+  uno.addComponent([
```
Comment (rohmanhm) [inline example/my-builder.js]: same like the above

File: src/client.js [hunk 3]
```diff
@@ -251,6 +245,7 @@ class UnoBuilder {
    */
   addComponent (url) {
     let type = 'component'
+    if (typeof url === 'string') url = [url]
```
Comment (rohmanhm) [inline src/client.js]: Here's the magic, user can passed string too.
But I'm not sure if they got preserved order results

**Comment by rohmanhm** [comment]
Closed.
Move to https://github.com/djavaweb/unobuilder/pull/32

**Comment by oknoorap** [comment]
async requests are needed when multiple authors create their own components, not only from single authors. their components load url with different domain.

You can imagine, there are author B that depends on author A components.

author B's component can't be added into canvas or it will looks ugly when author A's component is not loaded yet.

**Comment by rohmanhm** [comment]
@oknoorap oops, sorry I don't know about that. Maybe I'll keep this code into commit history

**Pull request closed without merging**